### ISSUE TITLE
NTR: Cystourethrography: ureteral orifices ectopic (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8920]
New term request Cystourethrography: ureteral orifices ectopic (elthat_100730011642):   UMLS: 2163952

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000413
- label: Ectopic ureteral orifice
- parents: [Abnormal ureter morphology (HP:0025633)](https://hpo.jax.org/app/browse/term/HP:0025633)
- pmid: PMID:2762191
- def: Drainage of the ureter on one or both sides into an abnormal location. Normally the ureter drains via the internal ureteral orifice at the trigone of the urinary bladder. Ectopic ureteral orifice can be either intravesical (bladder neck) or, more often, extravesical. Ectopia in women is more frequently found in the urethra and the vaginal vestibule; much less frequently in vagina and in uterus. In men it is mostly found in the posterior urethra; in the male genital tract is very rare.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8920
- synonyms:
